shared/actions/teams/index.js

```diff
@@ -443,6 +444,30 @@ function* _setupTeamHandlers(): SagaGenerator<any, any> {
   })
 }
 
+function* _badgeAppForTeams(action: Constants.BadgeAppForTeams) {
+  // TODO change newTeamIDs once the notification changes
+  const newTeams = action.payload.newTeamIDs || []
```

Comment:
Core will start sending us the teamnames here, so those will be inserted into the store instead of these IDs once that changes